db/errors.go

```diff
@@ -0,0 +1,14 @@
+package db
+
+import "errors"
+
+var ErrNotFound = errors.New("value not found")
+var ErrTransactionFailed = errors.New("value not found")
```

Comment:
should be something more like "transaction failed"

Reply:
Good catch. Fixed.